README.md

```diff
@@ -210,11 +210,17 @@ export default Ember.Controller.extend({
   session: Ember.inject.service('session'),
 
   actions: {
-    authenticate() {
+    async authenticate() {
       let { identification, password } = this.getProperties('identification', 'password');
-      this.get('session').authenticate('authenticator:oauth2', identification, password).catch((reason) => {
+      try {
+        await this.session.authenticate('authenticator:oauth2', identification, password);
+      } catch(e) {
         this.set('errorMessage', reason.error || reason);
```

Comment:
You missed converting the `reason` from the `.catch()` to the `catch (error) `.

```js
...
} catch (error) {
    this.set('errorMessage', error.error || error);
}

Reply:
I fixed it